sql/catalyst/src/main/scala/org/apache/spark/sql/catalog/v2/CatalogV2Implicits.scala

```diff
@@ -92,6 +92,10 @@ object CatalogV2Implicits {
     }
   }
 
+  implicit class NamePartsHelper(nameParts: Seq[String]) {
+    def toIdentifier: Identifier = Identifier.of(nameParts.init.toArray, nameParts.last)
```

Comment:
Why not add this to `MultipartIdentifierHelper`?

Reply:
`MultipartIdentifierHelper` takes a parameter called `namespace`. I'm a little worried about semantic mismatch here so I created a new one. How about we merge these two and call it `NamePartsHelper`? `nameParts` is very general.

Reply:
I think we could just change that one to use `nameParts` and combine the two.